docs/operation/security.md

```diff
@@ -1,16 +1,43 @@
 # Security
 
-Security is a critical aspect of any application. In this section, we will discuss the security features of the BanyanDB.
+Security is a critical aspect of any application. In this section, we will discuss the security features of the
+BanyanDB.
 
 ## Authentication
 
+### Username/Password Authentication
+
+BanyanDB supports username/password authentication for gRPC and HTTP server. The following flags are used to configure
+authentication:
+
+- `--auth-config-file`: Path to the authentication config file (YAML format).
+- `--hash-password`: It is used when enabling authentication for the first time. Encrypt the passwords in the provided
```

Comment:
You cannot use the flag. The configuration file should be updated by a third-party component, such as Vault. I suggest you use the plain password instead of the hash path.